.php_cs

```diff
@@ -18,5 +18,6 @@ return PhpCsFixer\Config::create()
         'array_syntax' => [
             'syntax' => 'short'
         ],
+        'no_unused_imports' => false,
```

Comment:
Why are we removing this?

Reply:
See https://github.com/mautic/mautic/pull/6991/files#r314940433